lib/vault.ts

```diff
@@ -184,3 +283,131 @@ export async function numberOfUnprocessedSubmissions(
     return allSubmissions.numberOfUnprocessedSubmissions;
   }
 }
+
+/**
+ * This method deletes all responses for a given form
+ * @param ability
+ * @param dynamoDb - DynamoDB Document Client
+ * @param formResponses - List of form submissions and confirmations
+ * @param formID
+ */
+export async function deleteResponses(
```

Comment:
Yeah I don't mind, I was thinking it could be used somewhere else in the future but I can just easily remove the export for now

Reply:
The main reason is to avoid having a function that should be used very carefully be available for any developer to call while it was originally created for a specific feature.

Reply:
I think this function is imported and used by the test suite. Would you be okay with keeping the export for this function and removing it for the others? @craigzour